benchmark.py

```diff
@@ -8,7 +8,7 @@
 import texttable
 import sys
 import codecs
-from platform import python_version_tuple
+from platform import python_version_tuple, platform
```

Comment:
Please revert this particular change as it is inconsistent with the principle of lazy loading - as is followed elsewhere in this project. What that means simply is that modules/packages/libraries should be imported as and when needed, and 'platform' is not needed at this line of code (it is used only within the 'benchmark' function)

Reply:
Alright, I'll fix it and push tomorrow. May I ask why you have decided to lazy load, it's quite unusual to do so (generally most python linters warn when imports are not on top)?